src/actions/types.ts

```diff
@@ -0,0 +1,12 @@
+export interface CoinGeckoPrice {
+  usd: number;
+}
+
+export interface CoinGeckoResponse {
```

Comment:
Nit: I found that making a generic Map<T> type with this format was really useful across my project was really helpful in reducing copy/paste typescript map stuff if you use maps in other locations too

Reply:
Cool idea, made that change here